build.gradle

```diff
@@ -115,3 +116,11 @@ task wrapper(type: Wrapper) {
     gradleVersion = '2.11'
     distributionUrl = "http://services.gradle.org/distributions/gradle-${gradleVersion}-all.zip"
 }
+
+if (JavaVersion.current().isJava8Compatible()) {
+    allprojects {
+        tasks.withType(Javadoc) {
+            options.addStringOption('Xdoclint:none', '-quiet')
+        }
+    }
+}
```

Comment:
I am sorry, I have overlooked your comment about that change. But please still open a separate PR for that .... thanks a lot.